{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,79 @@
+## For the Submitter
+Please make sure that you have followed all of the steps below, as
+it will help your PR be merged smoothly. Feel free to remove any
+sections that won't be relevant to reviewers of your PR, including this
+one.
+
+Anything marked with `TODO:` is something that you, the developer, should
+do as part of the PR.
+
+### Testing
+- TODO: Include unit, integration, acceptance tests as appropriate
```

Comment:
My issue with having TODO for these items is that it isn't clear how the reviewer will know that the developer has done them. Having a check box makes it clear that the developer asserts that they have completed that task, or that they know that they should but that they haven't got to it yet.

I suppose we could have a convention that the user switch the item from TODO to DONE or a check or something, but then that's what the GitHub tasks are for.

I like the TODO convention for something that the developer should consider before submitting the PR. They would then remove all TODO sections as part of filling in the cover letter.